test/run

```diff
@@ -11,10 +11,9 @@ TEST_SCENARIO=${TEST_SCENARIO:-verify}
 TEST_OS_DEFAULT=$(PYTHONPATH=bots python3 -c 'from machine.machine_core.constants import TEST_OS_DEFAULT; print(TEST_OS_DEFAULT)')
 
 case $TEST_SCENARIO in
-    verify|mobile|devel|firefox|firefox-devel)
+    verify|devel|firefox|firefox-devel)
```

Comment:
This "Remove support for mobile scenario" commit changes the mobile pixel tests, they now [dropped the tooltips](https://github.com/cockpit-project/pixel-test-reference/compare/dba18567f57691d3e0e4db3f3c1d5789f7a7853c..5e461db94d7f188eb42885e9c8ee583ee64735d2#diff-0ed3412b32e0734685f9414341d2c7dcad89d5873eb12c241015c8e60a5dcace). I wonder how that happened? Does changing the window size somehow change the mouse position?

(It's fine, I am just curious)

Reply:
My guess is that the tooltip appears only when the dialog is opened in a too narrow size.  Now the tests open the dialof in "desktop" layout and switch to "mobile" layout while it is already open.

My hunch is that this is buggy behavior of Patternfly, one way or the other.